www/nginx/src/opnsense/service/templates/OPNsense/Nginx/http.conf

```diff
@@ -171,8 +171,34 @@ server {
     satisfy {{ server.satisfy }};
 {% endif %}
     #include tls.conf;
+{% if server.errorpages is defined and server.errorpages != '' %}
+{%   for errorpage_uuid in server.errorpages.split(',') %}
+{%     set errorpage = helpers.getUUID(errorpage_uuid) %}
```

Comment:
`{% if errorpage is defined %}` to be added?

Reply:
I think every error page must have an uuid